README.md

```diff
@@ -0,0 +1,142 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="https://user-images.githubusercontent.com/65084923/236905326-efe684f3-a1d1-4acb-af4d-cf290f8b3982.png" alt="logo" width="100"  height="auto" />
+  <img src="https://user-images.githubusercontent.com/65084923/232604539-ea646b45-13a1-4214-9ffa-6720ec9634bf.png" alt="logo" width="100"  height="auto" />
+  <br/>
+  <img src="https://user-images.githubusercontent.com/65084923/220200035-01fa5e08-af6a-495b-b597-55157a2b7d95.gif" alt="logo" />
+  <br/>
+
+  <h3><b>Blog App - Ruby on Rails</b></h3>
+  <h3><b>Blog App - Ruby on Rails - ROR Microverse Project</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+
+- [💻 Getting Started](#getting-started)
+  - [Available Scripts](#available-scripts)
+  - [Clone](#clone)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 Blog App - Ruby on Rails - ROR Microverse Project<a name="about-project"></a>
+
+> Blog App - developed with Ruby on Rails.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://en.wikipedia.org/wiki/Ruby_(programming_language)">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Migration DB feature: tables and Relations**
+- **Models and Active Records**
+- **Rspec Test of all models**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and use as a template, follow these steps.
+
+### Clone <a name="clone"></a>
+
+Clone this repository to your desired folder:
+
+```sh
+  https://github.com/haftamudesta/Blog-App
+```
+
+To install the necessary dependencies, you should run:
+
+### `gem install bundler`
+
+### `bundler install`
+
```

Comment:
- [ ] [optioanl] It would be nice if you added instructions on how to run the app locally after cloning it.